Cloud Application Manager/DR Readiness/Create Node Pair.md

```diff
@@ -0,0 +1,51 @@
+{{{
+  "title": "Create Node Pair",
+  "date": "09-26-2019",
+  "author": "Sharon Wang",
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+
+**In this article:**
+
+* [Overview](#overview)
+* [Audience](#audience)
+* [Prerequisites](#prerequisites)
+* [Create a Node Pair](#create-a-node-pair)
+
+### Overview 
+
+This article is meant to assist users of Cloud Application Manager willing to use Disaster Recovery feature, trying to create new Replication Node Pairs
+
+### Audience
+
+All users of Cloud Application Manager who wants to use Disaster Recovery feature.
+
+### Prerequisites
+
+* An active *Cloud Application Manager* account
+* An existing *AWS* account configured in an [*AWS*](../Deploying Anywhere/using-your-aws-account.md) provider.
+* Both CLC and AWS providers are registered
+
+ 
+### Create a Node Pair
+1. Click on **Disaster Recovery** on the navigation bar, then click on the button **New** to create a new pair.
+![image](../../images/cloud-application-manager/dr-readiness/node_pair_1.png)
```

Comment:
```suggestion
![Create new node pair](../../images/cloud-application-manager/dr-readiness/node_pair_1.png)
```
This is what I was referring to when asking to update alternate texts to something more meaningful. Please, do so for the rest of the images.